src/components/common/IntroModal.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState, useEffect } from 'react';
+import { useTranslation } from 'next-i18next';
+import {
+  Button,
+  Modal,
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+  FormGroup,
+  Label,
+  Input,
+} from 'reactstrap';
+import styled from 'styled-components';
+
+const StyledButton = styled(Button)`
+  background-color: ${(props) => props.theme.brandDark};
+  color: ${(props) => props.theme.neutralLight};
+
+  &:hover {
+    background-color: ${(props) => props.theme.inputBtnFocusColor};
+    color: ${(props) => props.theme.neutralLight};
+  }
+`;
+
+const StyledLabel = styled(Label)`
+  &:hover {
+    color: ${(props) => props.theme.inputBtnFocusColor};
+    cursor: pointer;
+  }
+`;
+
+const StyledInput = styled(Input)`
+  &:checked {
+    background-color: ${(props) => props.theme.brandDark};
+    border-color: ${(props) => props.theme.brandDark};
+  }
+`;
+
+const StyledContainer = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
```

Comment:
Let's position this next to the button to make it more noticeable

```suggestion
  justify-content: flex-end;
  gap: 1rem;
```